apps/studio/src/server/modules/searchsg/searchsg.service.ts

```diff
@@ -0,0 +1,125 @@
+import wretch from "wretch"
+
+import { env } from "~/env.mjs"
+
+const SEARCHSG_BASE_URL = "https://api.services.search.gov.sg/admin"
+
+const ISOMER_UA = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) isomer"
+const SearchSgApi = {
+  Auth: `/v1/auth/token`,
+  App: `/v1/bootstrap/applications`,
+} as const
```

Comment:
suggestion: absolutely do not have to do this now, but I thought I quite like this pattern and maybe this can be extended somewhat, so it can be something like this:

```ts
const SearchSGApi = {
    Create: client.url("/v1/bootstrap/applications").post(),
    Update: client.url("/v1/bootstrap/applications").put(),
    ...
}
```

Thinking this might make the CRUD operations more obvious, cos right now the POST/PUT/GET is quite hidden down below.

Reply:
could you elaborate on this? i can do this down the line